app/controllers/patients_controller.rb

```diff
@@ -320,6 +316,10 @@ def move_to_household
       comment = "User added monitoree to a household. Monitoree with ID #{new_hoh_id} will now be responsible"\
                 ' for handling the reporting on their behalf.'
       History.monitoring_change(patient: current_patient, created_by: current_user.email, comment: comment)
+    else
+      # Create history item for new HoH
```

Comment:
This comment got separated from the code it applies to. Should be moved up into the `if` clause

Reply:
this should be addressed now